services/conference/src/scripts/components/map/BackgroundLayer/Background.tsx

```diff
@@ -6,29 +6,55 @@ import React from 'react'
 const SIZE = 5000
 
 const useStyles = makeStyles({
-  img: {
-    position: 'absolute',
-    transform: `translate(-${SIZE / 2}px, -${SIZE / 2}px)`,
-    backgroundColor: '#DFDBE5',
-    backgroundImage: `url("data:image/svg+xml,%3Csvg width='80' height='80' viewBox='0 0 80 80' xmlns='http://www.w3.org/2000/svg'%3E%3Cg fill='none' fill-rule='evenodd'%3E%3Cg fill='%239C92AC' fill-opacity='0.4'%3E%3Cpath d='M50 50c0-5.523 4.477-10 10-10s10 4.477 10 10-4.477 10-10 10c0 5.523-4.477 10-10 10s-10-4.477-10-10 4.477-10 10-10zM10 10c0-5.523 4.477-10 10-10s10 4.477 10 10-4.477 10-10 10c0 5.523-4.477 10-10 10S0 25.523 0 20s4.477-10 10-10zm10 8c4.418 0 8-3.582 8-8s-3.582-8-8-8-8 3.582-8 8 3.582 8 8 8zm40 40c4.418 0 8-3.582 8-8s-3.582-8-8-8-8 3.582-8 8 3.582 8 8 8z' /%3E%3C/g%3E%3C/g%3E%3C/svg%3E")`,
-    height: SIZE,
-    width: SIZE,
+  img: (props:BackgroundProps) => {
+    if (props.isTransparnet) {
+      return  {
+        position: 'absolute',
+        transform: `translate(-${SIZE / 2}px, -${SIZE / 2}px)`,
+        backgroundColor: 'rgba(0,0,0,0.1)',

```

Comment:
It seems the return objects are almost same, why not use `backgroundColor: props.isTransparent ? xxx : xxx`

Reply:
The best appearance is still not obvious.
It woll change more.
SVG is also a bit different.

Reply:
OK